### DISCUSSION TITLE
I made a utility to help add rtl_433 discovered devices to Home Assistant

### DISCUSSION THREAD

**Discussion by dewgenenny** [merbanan/rtl_433#2717] [Show and tell]
Hey, not sure if it's OK to post this here, I just wanted to share a utility I've been working on, to take the devices discovered by rtl_433 and (if you choose to) add them to Home Assistant.

I'd be super keen for your feedback / suggestions for improving... you can find it here: https://github.com/dewgenenny/rtl_433_discoverandsubmit/

rtl_433 is a totally epic project! Until last weekend I had no idea there was so many messages just floating around out there 🤣 Thanks to my neighbours I now have about 5 outdoor temperature sensors 🤣🤣🤣

**Comment by zuckschwerdt** [comment]
Sure. Add it to the wiki if you like. https://github.com/merbanan/rtl_433/wiki/Built-with-rtl_433%21

**Reply by dewgenenny** [reply]
Cool, thanks mate, have done :)
